persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java

```diff
@@ -221,7 +224,21 @@ public static String updateBallerinaToml(Path configPath, String module, String
                         moduleMembers = moduleMembers.add(member);
                     }
                 } else if (member instanceof TableArrayNode) {
-                    moduleMembers = moduleMembers.add(member);
+                    NodeList<KeyValueNode> fields = ((TableArrayNode) member).fields();
+                    for (KeyValueNode field : fields) {
+                        String value = field.value().toSourceCode().trim();
+                        if (field.identifier().toSourceCode().trim().equals(
+                                PersistToolsConstants.TomlFileConstants.KEYWORD_ARTIFACT_ID) &&
+                                (value).substring(1, value.length() - 1).equals(
+                                        PersistToolsConstants.TomlFileConstants.ARTIFACT_ID)) {
+                            moduleMembers = moduleMembers.remove(member);
```

Comment:
Why are we adding the members back? 

@sahanHe Are we updating the toml file even if we find the required configs? Why? We can simply break if both configurations are available cant we?

Reply:
No need for this as a new member node create to add the content to toml.